Lint check failing due missing translations

## Description

:app:lint check failing due missing translations

## Devices and Versions

CI

## Steps to reproduce

https://circleci.com/gh/RocketChat/Rocket.Chat.Android/5890?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

## Logs

```
> Task :app:lint FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:lint'.
> Lint found errors in the project; aborting build.
  
  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...
  
  Errors found:
  
  /home/circleci/project/app/src/main/res/values/strings.xml:174: Error: "action_share" is not translated in "es" (Spanish) [MissingTranslation]
      <string name="action_share">Share</string>
              ~~~~~~~~~~~~~~~~~~~
  /home/circleci/project/app/src/main/res/values/strings.xml:272: Error: "share_label" is not translated in "es" (Spanish) [MissingTranslation]
      <string name="share_label">Edit shared message</string>
              ~~~~~~~~~~~~~~~~~~
```

--- a/app/src/main/res/values-es/strings.xml
+++ b/app/src/main/res/values-es/strings.xml
@@ -38,6 +38,7 @@
     <string name="action_invisible">Invisible</string>
     <string name="action_drawing">dibujo</string>
     <string name="action_save_to_gallery">Guardar en la galería</string>
+    <string name="action_share">Compartir</string>
 
     <!-- Settings List -->
     <string-array name="settings_actions">
@@ -262,6 +263,7 @@
     <string name="header_unknown">Desconocido</string>
 
     <!--Notifications-->
+    <string name="share_label">Edita mensaje compartido</string>
     <string name="notif_action_reply_hint">RESPUESTA</string>
     <string name="notif_error_sending">La respuesta ha fallado. Inténtalo de nuevo.</string>
     <string name="notif_success_sending">Mensaje enviado a %1$s!</string>

